src/aap_eda/api/views/activation.py

```diff
@@ -120,7 +124,10 @@ def perform_destroy(self, activation):
         activation.status = ActivationStatus.DELETING
         activation.save(update_fields=["status"])
         logger.info(f"Now deleting {activation.name} ...")
-        delete_activation(activation_id=activation.id)
+        delete_activation(
```

Comment:
@bzwei Lets change the start/stop/delete/restart activation to start/stop/delete/restart rulebook_process.